eventhub/Module.md

```diff
@@ -1,540 +1,199 @@
-   map<string> brokerProps = {CorrelationId: "32119834", CorrelationId2: "32119834"};
-   map<string> userProps = {Alert: "windy", warning: "true"};
+    map<string> brokerProps = {CorrelationId: "32119834", CorrelationId2: "32119834"};
+    map<string> userProps = {Alert: "windy", warning: "true"};
 
-    eventhub:BatchEvent batchEvent = {
+    azure_eventhub:BatchEvent batchEvent = {
         events: [
             {data: "Message1"},
             {data: "Message2", brokerProperties: brokerProps},
             {data: "Message3", brokerProperties: brokerProps, userProperties: userProps}
         ]
     };
-    var result = eventHubClient->sendBatch("myhub", batchEvent, partitionId = 1);
-}
+    check eventHubClient->sendBatch("myhub", batchEvent, partitionId = 1);
 ```
-Sample is available at:
-https://github.com/ballerina-platform/module-ballerinax-azure.eventhub/blob/master/eventhub/samples/send_batch_event_to_partition.bal
 
-8. Sending a batch event with publisher id
+* Send a batch event with publisher id
 ```ballerina
-import ballerinax/azure.eventhub as eventhub;
-
-public function main() {
-   eventhub:ClientEndpointConfiguration config = {
-       sasKeyName: "<sas_key_name>",
-       sasKey: "<sas_key>",
-       resourceUri: "<resource_uri>"
-   };
-   eventhub:Client eventHubClient = checkpanic new (config);
-   map<string> brokerProps = {CorrelationId: "32119834", CorrelationId2: "32119834"};
-   map<string> userProps = {Alert: "windy", warning: "true"};
+    map<string> brokerProps = {CorrelationId: "32119834", CorrelationId2: "32119834"};
+    map<string> userProps = {Alert: "windy", warning: "true"};
 
-    eventhub:BatchEvent batchEvent = {
+    azure_eventhub:BatchEvent batchEvent = {
         events: [
             {data: "Message1"},
             {data: "Message2", brokerProperties: brokerProps},
             {data: "Message3", brokerProperties: brokerProps, userProperties: userProps}
         ]
     };
-    var result = eventHubClient->sendBatch("myhub", batchEvent, publisherId = "device-1");
-}
+    check eventHubClient->sendBatch("myhub", batchEvent, publisherId = "device-1");
 ```
-Sample is available at:
-https://github.com/ballerina-platform/module-ballerinax-azure.eventhub/blob/master/eventhub/samples/send_batch_event_with_publisherId.bal
 
-9. Create a new event hub
+* Create a new event hub
 ```ballerina
-import ballerinax/azure.eventhub as eventhub;
-
-public function main() {
-   eventhub:ClientEndpointConfiguration config = {
-       sasKeyName: "<sas_key_name>",
-       sasKey: "<sas_key>",
-       resourceUri: "<resource_uri>"
-   };
-   eventhub:Client eventHubClient = checkpanic new (config);
-   var result = eventHubClient->createEventHub("myhub");
-}
+   check eventHubClient->createEventHub("myhub");
 ```
-Sample is available at:
-https://github.com/ballerina-platform/module-ballerinax-azure.eventhub/blob/master/eventhub/samples/create_event_hub.bal
 
-10. Get an event hub
+* Get an event hub
```

Comment:
event hub -> Event Hub

Reply:
`Quick reference` section was removed after discussing with the team.